Bottom: 793b843eeca39b7aeb80aa899881640651f0c970
Top:    5f1a1373048f240063ebbfa0f32c67010b20d9b6
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2009-10-21 21:01:19 +0100

Refresh of shiny-3d-eye-candy

---

diff --git a/src/hid/gtk/gui-pinout-preview.c b/src/hid/gtk/gui-pinout-preview.c
index a086b19..8716836 100644
--- a/src/hid/gtk/gui-pinout-preview.c
+++ b/src/hid/gtk/gui-pinout-preview.c
@@ -210,16 +210,17 @@ ghid_pinout_preview_expose (GtkWidget * widget, GdkEventExpose * ev)
 
   glMatrixMode (GL_PROJECTION);
   glLoadIdentity ();
-  glOrtho (ev->area.x, ev->area.x + ev->area.width, ev->area.y + ev->area.height, ev->area.y, 0, 100);
+  glOrtho (ev->area.x, ev->area.x + ev->area.width, ev->area.y + ev->area.height, ev->area.y, -1000, 1000);
   glMatrixMode (GL_MODELVIEW);
   glLoadIdentity ();
-  glTranslatef (0.0f, 0.0f, -Z_NEAR);
+//  glTranslatef (0.0f, 0.0f, -Z_NEAR);
 
   glClearColor (gport->bg_color.red / 65535.,
                 gport->bg_color.green / 65535.,
                 gport->bg_color.blue / 65535.,
                 1.);
   glClearStencil (0);
+  glStencilMask (~0);
   glClear (GL_COLOR_BUFFER_BIT | GL_STENCIL_BUFFER_BIT);
 
   hidgl_reset_stencil_usage ();
